### PR TITLE
chore(docs): p-files Part1 - adjust headlines according to style guide + adjust some texts

### PR DESCRIPTION
**Pull request by codingCookie**
## Description
This PR introduces some changes to **first part** of the **p-files** in the **docs** folder.

#### What have i done?
- changed some headlines to fit the style guide
- a few text changes

## Related Issues
Related to https://github.com/gatsbyjs/gatsby/issues/18284

### CODE REVIEW

File: docs/docs/page-node-dependencies.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Page -> Node Dependency Tracking
+title: Page Node Dependency Tracking
```
Comment (LekoArts) [inline docs/docs/page-node-dependencies.md]: Please revert this change

File: docs/docs/pixabay-source-plugin-tutorial.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ to create the file using default options.
 
 ### Add dependencies
 
-You'll use a couple of modules from npm to add some helper functionality. Install them with:
+You'll use a couple of modules from `npm` to add some helper functionality. Install them with:
```
Comment (LekoArts) [inline docs/docs/pixabay-source-plugin-tutorial.md]: Please revert changes related to putting npm into a code block.

File: docs/docs/pixabay-source-plugin-tutorial.md [hunk 3]
```diff
@@ -377,7 +377,7 @@ You've written a local Gatsby plugin that:
 
 Congratulations!
 
-## Where next?
+## What's next?
```
Comment (LekoArts) [inline docs/docs/pixabay-source-plugin-tutorial.md]: Please revert this, too, as "Where next?" makes more sense here (saying "where can I look to go further"?)

File: docs/docs/production-app.md [hunk 4]
```diff
@@ -99,23 +99,23 @@ It's worth noting that the browser API runner is completely different to `api-ru
...
 
 This occurs in [loader.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js). The main function here is [getResourcesForPathname()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L314). Given a path, it will find its page, and import its component module json query results. But to do this, it needs access to that information. This is provided by [async-requires.js](/docs/write-pages/#async-requiresjs) which contains the list of all pages in the site, and all their dataPaths. [fetchPageResourcesMap()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L33) takes care of requesting that file, which occurs the [first time](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L292) `getResourcesForPathname()` is called.
 
-### window variables
+### Window variables
```
Comment (LekoArts) [inline docs/docs/production-app.md]: Please revert the change as `window` is defined as lowercase

Comment (muescha) [reply docs/docs/production-app.md]: Maybe add code block?

```suggestion
### `window` variables
```

File: docs/docs/pixabay-source-plugin-tutorial.md (hunk 2 above)
Comment (codingCookie) [reply docs/docs/pixabay-source-plugin-tutorial.md]: yes i can revert it - sure. 
i did it because a fews sentences above npm was in a code block - right under '### Create a `package.json` file'. 
what's the difference? or should we remove the ticks from the other npm sentence as well?

File: docs/docs/page-node-dependencies.md (hunk 1 above)
Comment (codingCookie) [reply docs/docs/page-node-dependencies.md]: 👍

File: docs/docs/production-app.md (hunk 4 above)
Comment (codingCookie) [reply docs/docs/production-app.md]: 👍

File: docs/docs/pixabay-source-plugin-tutorial.md (hunk 2 above)
Comment (LekoArts) [reply docs/docs/pixabay-source-plugin-tutorial.md]: npm shouldn't be in the code blocks, so I'd highly appreciate it if you could change these other occurrences in the file, too. Thanks! @codingCookie

Comment (codingCookie) [reply docs/docs/pixabay-source-plugin-tutorial.md]: 👍  removed all code-blocks regarding npm

File: docs/docs/page-creation.md [hunk 5]
```diff
@@ -8,14 +8,14 @@ A page is created by calling the [createPage](/docs/actions/#createPage) action.
...
 
 ## onCreatePage API
 
-Every time a page is created, plugins have the opportunity to handle its [onCreatePage](/docs/node-apis/#onCreatePage) event. This is used for things like creating `SitePage` nodes in [Internal Data Bridge](/docs/internal-data-bridge/), and for "path" related plugins such as [gatsby-plugin-create-client-paths](/packages/gatsby-plugin-create-client-paths/) and [gatsby-plugin-remove-trailing-slashes](/packages/gatsby-plugin-remove-trailing-slashes/).
+Every time a page is created, plugins have the opportunity to handle its [onCreatePage](/docs/node-apis/#onCreatePage) event. This is used for things like creating `SitePage` nodes in [internal data bridge](/docs/internal-data-bridge/), and for "path" related plugins such as [gatsby-plugin-create-client-paths](/packages/gatsby-plugin-create-client-paths/) and [gatsby-plugin-remove-trailing-slashes](/packages/gatsby-plugin-remove-trailing-slashes/).
```
Comment (marcysutton) [inline docs/docs/page-creation.md]: Internal Data Bridge is a proper noun, so I'd keep it as-is.

File: docs/docs/pixabay-source-plugin-tutorial.md [hunk 6]
```diff
@@ -28,15 +28,15 @@ _**NOTE:** if your data is local i.e. on your file system and part of your site'
 
 Your plugin is going to source images from Pixabay. You'll be able to configure your plugin in your site's `gatsby-config.js` file and write GraphQL queries to access your plugin's data.
 
-> **NOTE:** You'll need a Pixabay API key which you can get by [registering for a Pixabay account](https://pixabay.com/en/accounts/register/). Your API key is in the [“Search Images” section of the Pixabay API docs](https://pixabay.com/api/docs/#api_search_images).
+> **NOTE:** You'll need a Pixabay API key which you can get by [registering for a Pixabay account](https://pixabay.com/en/accounts/register/). Your API key is in the ["Search Images" section of the Pixabay API docs](https://pixabay.com/api/docs/#api_search_images).
```
Comment (marcysutton) [inline docs/docs/pixabay-source-plugin-tutorial.md]: Curly quotes aren't necessary to change – in fact they are more appropriate for bodies of text. Was there a specific reason to change this?

File: docs/docs/pixabay-source-plugin-tutorial.md [hunk 7]
```diff
@@ -362,9 +362,9 @@ Experiment with different options in your `gatsby-config.js` file to see how tha
 
 ## Publishing a plugin
 
-Don't publish this particular plugin to npm or the Gatsby Plugin Library, because it's just a sample plugin for the tutorial. However, if you've built a local plugin for your project, and want to share it with others, `npm` allows you to publish your plugins. Check out the npm docs on [How to Publish & Update a Package](https://docs.npmjs.com/getting-started/publishing-npm-packages) for more info.
+Don't publish this particular plugin to npm or the Gatsby Plugin Library, because it's just a sample plugin for the tutorial. However, if you've built a local plugin for your project, and want to share it with others, npm allows you to publish your plugins. Check out the npm docs on [How to publish & update a package](https://docs.npmjs.com/getting-started/publishing-npm-packages) for more info.
```
Comment (marcysutton) [inline docs/docs/pixabay-source-plugin-tutorial.md]: This references a page title, so it should probably remain in title case.

File: docs/docs/pixabay-source-plugin-tutorial.md [hunk 8]
```diff
@@ -362,9 +362,9 @@ Experiment with different options in your `gatsby-config.js` file to see how tha
...
 
-> **NOTE:** Once you have published your plugin on `npm`, don't forget to edit your plugin's `package.json` file to include info about your plugin. If you'd like to publish a plugin to the [Gatsby Plugin Library](/plugins/) (please do!), please [follow these steps](/contributing/submit-to-plugin-library/).
+> **NOTE:** Once you have published your plugin on npm, don't forget to edit your plugin's `package.json` file to include info about your plugin. If you'd like to publish a plugin to the [Gatsby plugin library](/plugins/) (please do!), please [follow these steps](/contributing/submit-to-plugin-library/).
```
Comment (marcysutton) [inline docs/docs/pixabay-source-plugin-tutorial.md]: This is also a page title, and should remain in title case.

File: docs/docs/porting-from-create-react-app-to-gatsby.md [hunk 9]
```diff
@@ -169,9 +169,9 @@ For more information about errors encountered during builds, see the doc on [deb
...
+For dynamic routes, you should implement routing with [@reach/router](https://reach.tech/router), which is already included with Gatsby. Dynamic routes can be implemented the same way you would implement a router in Create React App (or any other React application). However, because these routes won't be represented as HTML files in the final build, if you want users to be able to visit the routes directly (like entering the URL in the search bar), you'll need to generate pages in the `gatsby-node.js` file which is demonstrated in the [building apps with Gatsby](/docs/building-apps-with-gatsby/) guide.
```
Comment (marcysutton) [inline docs/docs/porting-from-create-react-app-to-gatsby.md]: Also a page title that should be in title case.

File: docs/docs/porting-from-create-react-app-to-gatsby.md [hunk 10]
```diff
@@ -211,7 +211,7 @@ Part of Gatsby's philosophy around tooling is [progressively disclosing complexi
 
 In terms of levels of abstraction, Gatsby allows you to move up or down to tap into more sophisticated, lower-level APIs without needing to eject like you would in Create React App.
 
-#### webpack
+#### Webpack
```
Comment (marcysutton) [inline docs/docs/porting-from-create-react-app-to-gatsby.md]: webpack is lowercased: https://webpack.js.org/branding/

File: docs/docs/progressive-web-app.md [hunk 11]
```diff
@@ -28,32 +28,32 @@ PWAs are apps delivered through the web (as opposed to native apps, which are pa
...
+Gatsby provides a plugin interface to add support for shipping a manifest with your site -- [gatsby-plugin-manifest](/packages/gatsby-plugin-manifest).
 
 ### It must implement a service worker.
 
 A [service worker](https://developers.google.com/web/fundamentals/primers/service-workers/) provides support for an offline experience for your site, and makes your site more resilient to bad network connections.
 
 It's a script that runs separately in the background, supporting features like push notifications and background sync.
 
-Gatsby also provides a plugin interface to create and load a service worker into your site -- [**gatsby-plugin-offline**](/packages/gatsby-plugin-offline).
+Gatsby also provides a plugin interface to create and load a service worker into your site -- [gatsby-plugin-offline](/packages/gatsby-plugin-offline).
```
Comment (marcysutton) [inline docs/docs/progressive-web-app.md]: Please keep bolding as-is.

File: docs/docs/pixabay-source-plugin-tutorial.md (hunk 6 above)
Comment (codingCookie) [reply docs/docs/pixabay-source-plugin-tutorial.md]: no specific reason - i just thought they are not "the right ones" ^^ because they look a little bit "at an angle"

File: docs/docs/progressive-web-app.md (hunk 11 above)
Comment (codingCookie) [reply docs/docs/progressive-web-app.md]: ok - i've changed it back. Is there a "rule" when a link should be bold and when not? i've changed it because no other link is bold
